starter-code/routes/passportRouter.js

```diff
@@ -1,22 +1,70 @@
-const express        = require("express");
-const router         = express.Router();
+const express = require("express");
+const router = express.Router();
 // User model
-const User           = require("../models/user");
+const User = require("../models/user");
 // Bcrypt to encrypt passwords
-const bcrypt         = require("bcrypt");
-const bcryptSalt     = 10;
+const bcrypt = require("bcrypt");
+const bcryptSalt = 10;
 const ensureLogin = require("connect-ensure-login");
-const passport      = require("passport");
+const passport = require("passport");
 
 
+router.get("/login", (req, res, next) => {
+    res.render("passport/login", { "message": req.flash("error") });
```

Comment:
That's not how `req.flash` works. What that method does is setting in the `messages` variable in your views  (which is an object) a property which name is the first provided parameter and which value is the second one.

For example if you do `req.flash('info', 'I am a flash message')` you can do `<%= messages.info %>` in any view, which means that the resulting html for that specific request will contain *I am a flash message*.